.vscode/extensions.json

```diff
@@ -3,6 +3,9 @@
         "ms-python.black-formatter",
         "ms-python.flake8",
```

Comment:
Since you seem to be on-fire with pre-commit hooks and styles, maybe it is time to make a decision on continuing with flake8 or moving to ruff, which we already use in gridlington and WSIMOD.